pom.xml

```diff
@@ -103,26 +103,27 @@
     </developers>
 
     <properties>
-        <kotlin.version>1.5.21</kotlin.version>
+        <kotlin.version>1.7.10</kotlin.version>
         <junit.version>4.12</junit.version>
         <muleWsdl4j.version>1.6.4</muleWsdl4j.version>
         <slf4j.version>1.7.29</slf4j.version>
         <apache.commons.lang3.version>3.12.0</apache.commons.lang3.version>
-        <cxfRtWsdlVersion>3.4.4</cxfRtWsdlVersion>
+        <cxfRtWsdlVersion>3.5.3</cxfRtWsdlVersion>
         <hamcrest.version>2.2</hamcrest.version>
-        <mule.metadata.version>1.2.4</mule.metadata.version>
+        <mule.metadata.version>1.5.0-SNAPSHOT</mule.metadata.version>
```

Comment:
Be careful. Did you check if is accurate to change libraries to manage the metadata in a support branch? What changes were introduced in the previous minor versions?